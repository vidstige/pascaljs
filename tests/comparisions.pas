program comparisions;
var
    x: Integer;
    y: Integer;
    z: Boolean;
begin
    x := 17;
    y := 99;
    WriteLn('x = y  ', x=y);
    WriteLn('x >= y ', x>=y);
    WriteLn('x <= y ', x<=y);
    WriteLn('x <> y ', x<>y);
    WriteLn('x > y  ', x>y);
    WriteLn('x < y  ', x<y);
end.