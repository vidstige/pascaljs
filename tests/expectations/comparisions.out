x = y  false
x >= y false
x <= y true
x <> y true
x > y  false
x < y  true
